sample/src/js/index.js

```diff
@@ -1,44 +1,71 @@
 import NetworkTest from 'opentok-network-test-js';
-import createChart from './chart.js';
+import { createChart, createRtcStatsChart } from './chart.js';
 import * as ConnectivityUI from './connectivity-ui.js';
 import config from './config.js';
+
 let sessionInfo = config;
 let otNetworkTest;
 let audioOnly;
 
 const precallDiv = document.getElementById('precall');
-precallDiv.querySelector('#precall button').addEventListener('click', function () {
-    document.getElementById('connectivity_status_container').style.display = 'block';
-    precallDiv.style.display = 'none';
-    startTest();
-})
+precallDiv.querySelector('#precall button').addEventListener('click', () => {
+  document.getElementById('connectivity_status_container').style.display = 'block';
+  precallDiv.style.display = 'none';
+  startTest();
+});
 
 function startTest() {
-    audioOnly = precallDiv.querySelector('#precall input').checked;
-    var timeoutSelect = precallDiv.querySelector('select');
-    var timeout = timeoutSelect.options[timeoutSelect.selectedIndex].text * 1000;
-    var options = {
-        audioOnly: audioOnly,
-        timeout: timeout
-    };
-    otNetworkTest = new NetworkTest(OT, sessionInfo, options);
-    otNetworkTest.testConnectivity()
-        .then(results => ConnectivityUI.displayTestConnectivityResults(results))
-        .then(testQuality);
+  audioOnly = precallDiv.querySelector('#precall input').checked;
+  const timeoutSelect = precallDiv.querySelector('select');
+  const timeout = timeoutSelect.options[timeoutSelect.selectedIndex].text * 1000;
+  const options = {
+    audioOnly,
+    timeout,
+  };
+    
+  otNetworkTest = new NetworkTest(OT, sessionInfo, options);
+  otNetworkTest.testConnectivity()
+    .then((results) => ConnectivityUI.displayTestConnectivityResults(results))
+    .then(testQuality);
 }
 
 function testQuality() {
     createChart('audio');
     createChart('video');
+
+    createRtcStatsChartForAudio('audioBitrateChart', 'bitrate', 'Audio bitrate over time');
+    createRtcStatsChartForAudio('audioByteSentChart', 'byteSent', 'Audio bytes sent over time');
+    createRtcStatsChartForVideo('availableOutgoingBitrateChart', 'availableOutgoingBitrate', 'Available outgoing bitrate over time');
+    createRtcStatsChartForVideo('videoStats', 'Total bitrate video kb/s', 'Video kb/s');
+    createRtcStatsChartForVideo('videoStatsFramerate', 'framerate', 'Video Framerate');
+    createRtcStatsChartForVideo('videoStatsPliCount', 'pliCount', 'Video PLI Count');
```

Comment:
How should developers use the new stats (like PLI count)? Please add info to the readme.